### DISCUSSION TITLE
How to create single page react app with turbopack

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [vercel/turborepo#2989] [Help] [answered]
Hi. Is there any example to create react single page app with turbopack?

**Comment by anthonyshew** [comment, marked as answer]
The Turbopack team is using the Next.js Compiler as a proving ground to deliver better compiling to the 800,000+ Next.js users out there.

Once we have confidence that Turbopack is stable there, plans are to make it a general-purpose bundler. Stay tuned!
